### PR TITLE
Add onboarding warning

### PR DESCRIPTION
**Pull request by lucien-george**
As of January, applicants might receive an invitation email before the start of their batch to get onboarded on kitt if the admission manager decides to do so. Therefore we added a warning message in the setup to make sure they don't onboard themselves twice on Kitt

### CODE REVIEW

**Review by Eschults — APPROVED** [review]
Great 👍 

To merge with https://github.com/lewagon/setup/pull/220 and https://github.com/lewagon/kitt/pull/4105

**Comment by barangerbenjamin** [comment]
LGTM but the diff in `macOs.md`, `ubuntu.md` & `windows.md` reflects more changes than your modify partial?

File: LINUX.md [hunk 1]
```diff
@@ -480,7 +480,7 @@ sudo service docker stop
 
 ## Ubuntu inotify
 
-Ubuntu is always tracking changes in your folders, and to do this it uses inotify.
+Ubuntu is always tracking changes in your folders and to do this it uses inotify.
```
Comment (barangerbenjamin) [inline LINUX.md:483]: This change does not come from your partial

File: WINDOWS.md [hunk 2]
```diff
@@ -187,7 +187,7 @@ For many computers, this is already the case. Let's check:
   - The option can be called differently according to your computer:
       - Intel: `Intel VT-x`, `Intel Virtualization Technology`, `Virtualization Extensions`, `Vanderpool`...
       - AMD: `SVM Mode` or `AMD-V`
-  - Save the changes after activation, and reboot the computer through the appropriate option
+  - Save the changes after activation and reboot the computer through the appropriate option
```
Comment (barangerbenjamin) [inline WINDOWS.md:190]: same

File: WINDOWS.md [hunk 3]
```diff
@@ -235,30 +235,30 @@ If you are not familiar with video calls, here is a great [article](https://mart
 ### Install WSL 1
 WSL is the development environment we are using to run Ubuntu. You can learn more about WSL [here](https://docs.microsoft.com/en-us/windows/wsl/faq).
 
-We will install WSL through the Powershell Terminal:
+We will install WSL through the PowerShell Terminal:
```
Comment (barangerbenjamin) [inline WINDOWS.md:238]: same

**Pull request merged**